### PR TITLE
Update nav.json

### PR DESCRIPTION
**Pull request by Hugo6991**

### CODE REVIEW

File: static/locales/zh-TW/nav.json [hunk 1]
```diff
@@ -3,5 +3,9 @@
   "nav.solutions": "解決方案",
   "nav.developer": "開發者",
   "nav.pricing": "價格",
-  "nav.more": "更多"
-}
\ No newline at end of file
+  "nav.more": "更多",
+  "nav.community":"社群",
+  "nav.Token":"代幣",,
```
Comment (chihsuan) [inline static/locales/zh-TW/nav.json]: Could you remove the additional `,`?

**Comment by Hugo6991** [comment]
``

File: static/locales/zh-TW/nav.json (hunk 1 above)
Comment (Hugo6991) [reply static/locales/zh-TW/nav.json]: >   "nav.more": "更多",
>   "nav.community":"社群",
>   "nav.Token":"代幣",

Like this?

**Comment by Hugo6991** [comment]
"nav.more": "更多",
  "nav.community":"社群",
  "nav.Token":"代幣",

File: static/locales/zh-TW/nav.json (hunk 1 above)
Comment (chihsuan) [reply static/locales/zh-TW/nav.json]: yes

Comment (chihsuan) [reply static/locales/zh-TW/nav.json]: You could use this online tool to validate the format. https://jsonlint.com/

**Pull request closed without merging**